cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTable.java

```diff
@@ -243,6 +276,19 @@ private Put getIncrementalPut(Put existing, byte[] row) {
       .build();
   }
 
+  private IncrementBuilder getIncrement(IncrementBuilder existing, byte[] row, boolean transactional)
+    throws IOException {
+    if (existing != null) {
```

Comment:
That's the sad truth. We should have test case that having multiple rows updating the same column, committing in the same tx.